packages/fxa-customs-server/lib/server.js

```diff
@@ -120,6 +134,33 @@ module.exports = async function createServer(config, log) {
     throw err;
   }
 
+  function reportMetrics(request) {
+    const path = request._route.path.replace(/^\//, '').replace(/\//g, '_');
```

Comment:
I'd prefer to use strings to search instead of regexes.

Reply:
Updated to do this how auth-server does